lib/meilisearch-rails.rb

```diff
@@ -887,7 +887,7 @@ def ms_configurations
 
     def ms_primary_key_method(options = nil)
       options ||= meilisearch_options
-      options[:id] || options[:primary_key] || :id
+      options[:primary_key] || options[:id] || :id
```

Comment:
For what this fix is for? :)

Reply:
`primary_key `should go before `id` since it's the terminology used by MeiliSearch